src/pathfinding_service/model/token_network.py

```diff
@@ -210,6 +210,10 @@ def to_dict(self) -> dict:
         assert self.fees is not None
         return dict(
             path=[to_checksum_address(node) for node in self.nodes],
+            matrix_users=[
+                self.reachability_state._address_to_userids[node]  # pylint: disable=W0212
```

Comment:
I believe there can be some race conditions hidden in here. I think this should at least be guarded against `KeyError`s.

Reply:
I'll return a descriptive 500 in that case, now.